docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/entity-name-is-not-unique.md

```diff
@@ -14,9 +14,9 @@ related:
     link: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-method-is-overridden-on-the-project-level.html
 ---
 
-Modules have public and private APIs. While public API updates always support backward compatibility, private API updates can break backward compatibility. So, backward compatibility is not guaranteed in the private API. For example, if you use a core method on the project level, and it is updated or removed with an update, it can cause unexpected issues.
+The entity name is not unique. The page describes why entities on the project side have to have a unique name (containing some prefix). The project has to follow the rule to be able to correctly receive Spryker autoupdates.
```

Comment:
First sentence just duplicates the title of the page. I think it's better to start with successful scenario, so something like: 

```
All entities in your project should have the unique names. The easiest way to ensure that is to use a prefix in entities' names. Unique entities names are required for successful delivery of Spryker autoupdates to your project.
```

But @andriitserkovnyi can formulate it better

Reply:
Thanks 
Replaced with the suggested structure